image_provisioner/playbooks/roles/pbench-kickstart/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: create directory to store pbench results, store openshift templates for pbench
+  file: 
+    path: "{{ item }}"
+    state: directory
+  with_items:
+     - /var/lib/pbench-agent
+     - /var/home/openshift-templates/pbench
+
+- name: copy openshift template for pbench
+  copy: 
+    src: pbench-agent-daemonset.yml  
+    dest: /var/home/openshift-templates/pbench 
+    owner: root 
+    group: root 
+    mode: 0644
+     
+# not using docker_image module of ansible since that requires docker-py package to be installed on the atomic host
+- name: pull pbench-agent from dockerhub
+  command: docker pull ravielluri/image:agent
```

Comment:
Might be okay for now, but I don't think we want tooling to require dockerhub images.

Reply:
Open to other ideas.  Ultimately we need a registry that's available both internally and on AWS.  And last I asked, we weren't allowed to co-opt the ops registry for our stuff.  Unfortunately I suspect no better answer than dockerhub (alternative would be to build the images every time image provisioner gets kicked off).  Thoughts?